mirrord-schema.json

```diff
@@ -287,6 +287,17 @@
           "format": "uint64",
           "minimum": 0.0
         },
+        "tolerations": {
+          "title": "agent.tolerations {#agent-tolerations}",
+          "description": "Add extra pod tolerations. (not with ephemeral agents)",
```

Comment:
Set tolerations (overrides default tolerate everything, set to an empty array to have no tolerations at all)